installers/installer.iss

```diff
@@ -55,9 +55,8 @@ end;
 
 [Files]
 Source: "dist\sasview\sasview.exe"; DestDir: "{app}"; Flags: ignoreversion
+Source: "dist\sasview\plugin_models\*"; DestDir: "{%USERPROFILE}\.sasview\plugin_models"
```

Comment:
Is this a separate change for the windows installer?

Reply:
This was branched off a previous branch that is already merged. The only changes unique to this branch are in `ci.yml`. For some reason, the remaining changes still appear.